src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php

```diff
@@ -65,6 +65,23 @@ private function hasDefaultValue(\ReflectionParameter $parameter)
     }
 
     /**
+     * Returns if the argument is allowed to be null but is still mandatory.
+     *
+     * @param \ReflectionParameter $parameter
+     *
+     * @return bool
+     */
+    private function isNullable(\ReflectionParameter $parameter)
+    {
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
I use this feature because that's what is being used everywhere to be honest. Also a method check causes a bit more overhead than this equation. Maybe there's a `if HHVM and method exists`? Maybe the feature simply exists already